docs/reference/rollup/understanding-groups.asciidoc

```diff
@@ -7,7 +7,7 @@ experimental[]
 
 To preserve flexibility, Rollup Jobs are defined based on how future queries may need to use the data.  Traditionally, systems force
 the admin to make decisions about what metrics to rollup and on what interval.  E.g. The average of `cpu_time` on an hourly basis.  This
-is limiting; if, at a future date, the admin wishes to see the average of `cpu_time` on an hourly basis _and partitioned by `host_name`_,
+is limiting, in the future if the admin wishes to see the average of `cpu_time` on an hourly basis and partitioned by `host_name`,
```

Comment:
The suggested change isn't grammatically correct. Can you explain the problem you were attempting to solve with this?

Reply:
Hi @jrodewig this is the original doc:
https://www.elastic.co/guide/en/elasticsearch/reference/master/rollup-understanding-groups.html
I think:
1. There should no comma after "if".
2. "host_name" format is not correct.
![image](https://user-images.githubusercontent.com/15325213/102508652-97d2b880-40c0-11eb-8f0c-0da5288ba151.png)
![image](https://user-images.githubusercontent.com/15325213/102508893-de281780-40c0-11eb-9af4-0c1e92d4a2d0.png)